views/index.html

```diff
@@ -3,44 +3,14 @@
 <html lang="en-GB">
 
   <head>
-    <meta charset="utf-8">
-    <meta name="application-name" content="Echidna">
-    <meta name="description" content="W3C's orchestrator for the new publication workflow">
-    <meta name="keywords" content="w3c,publication,workflow">
-    <meta name="author" content="W3C">
+    <meta charset="utf-8" />
+    <meta name="application-name" content="Echidna" />
+    <meta name="description" content="W3C's orchestrator for the new publication workflow" />
+    <meta name="keywords" content="w3c,publication,workflow" />
+    <meta name="author" content="W3C" />
```

Comment:
Actually @tripu I removed them myself so that I can merge directly. No need to wait for another TZ round trip... Hope you don't mind! :)